include/z64player.h

```diff
@@ -5,6 +5,14 @@
 
 struct Player;
 
+typedef enum {
+    /* 0x00 */ PLAYER_SWORD_NONE,
+    /* 0x01 */ PLAYER_SWORD_KOKIRI,
+    /* 0x02 */ PLAYER_SWORD_MASTER,
+    /* 0x03 */ PLAYER_SWORD_LONG,
+    /* 0x04 */ PLAYER_SWORD_MAX
+} PlayerSword;
```

Comment:
As far as I know, there is nothing to consider as player sword values. The other "equipment" enums dedicated to player are associated to `currentShield`/`currentTunic`/`currentBoots`/`currentMask` in the Player struct (not the actual equip values stored in save context, which is a very important difference), but `currentSword` is just the item ID so there is no use for a player sword enum.